centreon/src/Security/Domain/Authentication/AuthenticationService.php

```diff
@@ -143,7 +144,7 @@ public function findAuthenticationTokensByToken(string $token): ?AuthenticationT
     /**
      * @inheritDoc
      */
-    public function updateAuthenticationTokens(AuthenticationTokens $authenticationTokens): void
+    public function updateAuthenticationTokens(SecurityAuthenticationTokens|AuthenticationTokens $authenticationTokens): void
```

Comment:
Please create an interface instead

Reply:
done, i fix it with interface already exicte so is done